CONTRIBUTING.md

```diff
@@ -82,12 +84,13 @@ to fix the issues) execute the second available CMake target - run a command:
 
```

Comment:
the above `execute the second available CMake target` sentence should be rephrased (we have now 4 checks)